components/CMakeLists.txt

```diff
@@ -51,7 +51,11 @@ add_component_dir (shader
 add_component_dir (sceneutil
     clone attach visitor util statesetupdater controller skeleton riggeometry morphgeometry lightcontroller
     lightmanager lightutil positionattitudetransform workqueue unrefqueue pathgridutil waterutil writescene serialize optimizer
+<<<<<<< HEAD
     actorutil detourdebugdraw navmesh agentpath shadow mwshadowtechnique occlusionquerynode
+=======
+    actorutil detourdebugdraw navmesh agentpath shadow mwshadowtechnique vismask recastmesh
+>>>>>>> 2a31382e2014a79cf6f2e4a124239edf9e10e85e
```

Comment:
yuk...

Reply:
dont know by which magic it achieved to build